lib/console.js

```diff
@@ -454,10 +450,38 @@ Console.prototype.table = function(tabularData, properties) {
   return final(keys, values);
 };
 
-module.exports = new Console({
+function noop() {}
+
+// See https://console.spec.whatwg.org/#console-namespace
+// > For historical web-compatibility reasons, the namespace object
+// > for console must have as its [[Prototype]] an empty object,
+// > created as if by ObjectCreate(%ObjectPrototype%),
+// > instead of %ObjectPrototype%.
+
+// Since in Node.js, the Console constructor has been exposed through
+// require('console'), we need to keep the Console constructor but
+// we cannot actually use `new Console` to construct the global console.
+// Therefore, the console.Console.prototype is not
+// in the global console prototype chain anymore.
+const consolePrototype = Object.create(Object.prototype);
```

Comment:
Isn't `Object.create(Object.prototype)` just `{}` ?

Reply:
I don't have an opinion on this, it just somehow seems more readable to be if we just do exactly what the spec says (but probably `{}` is more readable for people who skip the comments).